COMPOUND_COMPONENTS.md

```diff
@@ -0,0 +1,466 @@
+# Compound Components
+
+## What is a compound component?
+
+Compound components is a pattern in which components are used together in such a way that describes a higher level of abstraction. A desktop computer is composed of many components that can be serviced and configured individually. Similarly, a compound component allows simultaneous configuration through access to each sub-component. Here are the parts of a compound component:
+
+* [Container component](#container-components)
+* [Sub-components](#sub-components)
+* [Shared model (optional, advanced)](#models)
+* [Behavior hooks (optional, advanced)](#behavior-hooks)
+
+A compound component contrasts with a configuration component which instead configures from a single interface. A configuration component might be like choosing a desktop computer based on stats - how much RAM or how fast the CPU should be or based on what you want to use it for. A compound component is more like buying the parts individually and assembling yourself.
```

Comment:
Consider a slight expansion to: "A compound component contrasts with a configuration component which instead configures from a single interface." ...with "... a single interface, like a configuration object or multiple props in React."

Reply:
I like the PC analogy!

Reply:
Do you mean like this?

```suggestion
A compound component contrasts with a configuration component which instead configures from a single interface, like a configuration object or multiple props in React. A configuration component might be like choosing a desktop computer based on stats - how much RAM or how fast the CPU should be or based on what you want to use it for. A compound component is more like buying the parts individually and assembling yourself.
```